config/config.json

```diff
@@ -1,28 +1,19 @@
 {
   "server": {
     "host": "127.0.0.1",
-    "port": 3000
+    "port": 4000
   },
   "solvers": {
     "example-solver": {
       "solver_id": "example-solver",
       "adapter_id": "oif-v1",
-      "endpoint": "https://api.example.com/v1",
+      "endpoint": "http://127.0.0.1:3000/api",
       "timeout_ms": 1500,
       "enabled": true,
       "max_retries": 1,
       "headers": null,
-      "name": "Example Solver",
-      "description": "Example Solver Description",
-      "supported_assets": [
```

Comment:
supported_assets fields can be defined but it acts as an override for values. 

By default when not set we will automatically fetch supported assets from solver.